en/docs/learn/ai/rag/build-a-rag-application.md

```diff
@@ -1,18 +1,20 @@
 # Build a RAG Application
+This tutorial guides you through creating a Retrieval-Augmented Generation (RAG) system using the Ballerina Integrator. While there are several ways to structure a RAG workflow, we’ll focus on a typical two-phase approach: ingestion and retrieval.
 
-This tutorial guides you through creating a Retrieval-Augmented Generation (RAG) system using the Ballerina Integrator. While there are several ways to structure a RAG workflow, we'll focus on a typical two-phase approach.
-
-## RAG overview
-
-### Data ingestion 
+## RAG ingestion 
 This step is managed through [Devant](https://wso2.com/devant/docs/ai/rag-application/) and it focuses on preparing documents for efficient retrieval in the RAG system.
 
 - Chunk the information into smaller, meaningful sections
 - Convert each chunk into embeddings using an embedding model
 - Store embeddings in the vector database for efficient retrieval
 
-### Data retrieval 
-This tutorial focuses on implementing the data retrieval component of a Retrieval-Augmented Generation (RAG) system using the [Ballerina Integrator](https://wso2.com/integrator/bi/).
+We assume you've already run the Devant to handle document processing and create your vector index. Devant manages the complete ingestion independently from our main application flow. The Integrator will focus solely on the rag retrieval part, using the pre-created vector index from Devant's ingestion.
+
+!!! note  
+    If you haven't completed the above steps, follow this detailed tutorial [video](https://www.youtube.com/watch?v=8GlrHYS-EYI&list=PLp0TUr0bmhX4colDnjhEKAnZ3RmjCv5y2&ab_channel=WSO2) to understand how to set up the RAG ingestion and create your vector index.
+
```

Comment:
do we have to link the video here also? We can move this to devant docs, if it is already not there.